### ISSUE TITLE
Does goodmoney go in its own routine? 

### ISSUE THREAD

**Issue by jgerardsimcock** [ClimateImpactLab/prototypes#7]
I am not sure how to think of goodmoney. Is it something specific to mortality?

**Comment by delgadom** [comment]
no. but it's also not something that is done in every case. I think in general we should develop a way of specifying optional, but common, operations, such as goodmoney, adaptation scenarios, and flat-curve clipping. Perhaps these should be stand-alone functions? Mixins? Impact initialization options?
